docs/pages/other-env.mdx

```diff
@@ -0,0 +1,193 @@
+# Tooling for other environments
+
+The main client-side feature of MUD is _synchronization_, having a local on-client copy of the up-to-date state of the `World` object.
+To support other environments, such as [Unity](https://unity.com/) or [Unreal Engine](https://www.unrealengine.com/), this is the main functionality that needs to be ported.
+
+The initial synchronization is best done using the [Indexer](indexer) to get the starting state.
+The Indexer is written in TypeScript, but it can be [accessed through HTTP](./indexer#using-http)
+
+Once the client is in sync, it can listen to blockchain events emitted by the `World` to update the local copy of the state as needed.
+This is a lot more efficient than reading the entire state of the world from the Indexer repeatedly.
+
+## Reading and interpreting MUD events
+
+### Data events
+
+Data events emitted by a `World` only have a single topic, the event type, and these parameters:
+
+- `bytes32 table`, which is a single 32 byte value that contains both the namespace and the tableID.
```

Comment:
```suggestion
- `bytes32 table`, which is a single 32 byte value that contains both the namespace and the tableId.
```
we generally go with `Id`/`id` over `ID`

Reply:
since this is prose and not code, I would change this to "table ID"